src/ci/docker/host-aarch64/aarch64-gnu/Dockerfile

```diff
@@ -26,4 +26,6 @@ ENV RUST_CONFIGURE_ARGS \
  --enable-sanitizers \
  --enable-profiler \
  --enable-compiler-docs
-ENV SCRIPT python3 ../x.py --stage 2 test
+# FIXME: Skipping cargo panic_abort_doc_tests due to https://github.com/rust-lang/rust/issues/123733
+ENV SCRIPT python3 ../x.py --stage 2 test && \
+  python3 ../x.py --stage 2 test src/tools/cargo --test-args "--skip panic_abort_doc_tests"
```

Comment:
```suggestion
  python3 ../x.py --stage 2 test src/tools/cargo --test-args \"--skip panic_abort_doc_tests\"
```

Reply:
Thanks! I could have sworn I tested this locally, but maybe I only ran it without the skip to re-confirm the issue, and forgot to run again...